PowerEditor/src/WinControls/ProjectPanel/ProjectPanel.cpp

```diff
@@ -784,6 +1289,64 @@ HTREEITEM ProjectPanel::addFolder(HTREEITEM hTreeItem, const TCHAR *folderName)
 	return addedItem;
 }
 
+HTREEITEM ProjectPanel::addDirectory(HTREEITEM hTreeItem, const TCHAR *folderName, bool root, const TCHAR *monitorPath, const std::vector<generic_string>* filters)

```

Comment:
> Imho it probably should be better consistent currently, and refactored in a separate commit for all occurrences?

Normally, yes..... But this is *new* code and the `Notepad++` devs are, at times, quite hostile to code cleanup/refactoring.